src/cli/commands/generate.js

```diff
@@ -30,6 +30,14 @@ const utils = require('../../utils');
 handlebars.registerHelper('slugify', (str) => string(str).slugify().s);
 handlebars.registerHelper('trim', (str) => string(str).trim().s);
 handlebars.registerHelper('release_quality', utils.createReleaseQualityBadge);
+handlebars.registerHelper('if_eq', (left, right, opts) => {
+  if (left === right) {
+    return opts.fn(this);
+  }
+  else {
```

Comment:
Remove the `else` block and simply have `return opts.inverse(this);` be the last line of the function.

Reply:
Why? Isn't `if_eq`/`else` useful in a template?

Reply:
Yes, keep the helper function. I'm just saying change the code to:

```js
handlebars.registerHelper('if_eq', (left, right, opts) => {
  if (left === right) {
    return opts.fn(this);
  }
  return opts.inverse(this);
});
```

Reply:
Oh I see. Okay.